user_deletion/apps.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+
+class UserDeletionConfig(AppConfig):
+    name = 'user_deletion'
+
+    # users are notificed after 12 months of inactivity
+    MONTH_NOTIFICATION = 12
+    # users are deleted after 13 months
+    MONTH_DELETION = 13
```

Comment:
That would probably be a separate PR, right?  Not a difficult change conceptually, but enough of a faff that I think we can do it elsewhere.